src/components/Person/Person.jsx

```diff
@@ -1 +1,36 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+export const Person = ({ person }) => {
+  const maritalStatus = person.sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+
+      {person.age && (
+        <p className="Person__age">{`I am ${person.age}`}</p>
+      )}
```

Comment:
Use destructuring for getting access to values of props object
`
    const {
    name,
    age,
    sex,
    isMarried,
    partnerName,
  } = person;
`